cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -93,3 +95,67 @@ def create_sketch(ctx, name, description):
         description = name
     sketch = api_client.create_sketch(name=name, description=description)
     click.echo(f"Sketch created: {sketch.name}")
+
+
+@sketch_group.command("export", help="Export a sketch")
+@click.option("--filename", required=True, help="Filename to export to.")
+@click.pass_context
+def export_sketch(ctx, filename):
+    """Export a sketch to a file.
+
+    Args:
+        filename: Filename to create
+    """
+    sketch = ctx.obj.sketch
+    click.echo("Executing export . . . ")
+    click.echo("Depending on the sketch size, this can take a while")
+    # start counting the time the export took
+    start_time = time.time()
+    try:
+        search_obj = search.Search(sketch=sketch)
+
+        click.echo(f"Number of events in that sketch: {search_obj.expected_size}")
+
+        search_obj.to_file(filename)
+        # Using the sketch.export function might also work
+        # TODO: https://github.com/google/timesketch/issues/2344
+        # sketch.export("./test.zip")
```

Comment:
Does it work or not? Maybe we should be a bit more clear on the message here :)

Reply:
I did some tests and with more than 10 mil events on a local system it timed out, but this is not related to the CLI client